apps/test/unit/templates/projects/PersonalProjectsTableTest.js

```diff
@@ -8,7 +8,8 @@ import {stubFakePersonalProjectData} from '@cdo/apps/templates/projects/generate
 import {publishMethods} from '@cdo/apps/templates/projects/projectConstants';
 
 describe('PersonalProjectsTable', () => {
-  it('renders a table', () => {
+  // In the fake data used, the recency of the projects' updatedAt field is consistent with the numbering in the name; for example, the project named "Personal Project 1" has the most recent updatedAt time.
```

Comment:
hmmm, upon further contemplation I realized I don't actually like this consistency because it doesn't ensure that the projects are being sorted by recency - they could just be rendering in the order they are in list passed in.